blazor/getting-started/blazor-webassembly-authentication.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: Blazor WebAssembly App with Authentication Library | Syncfusion
+description: Check out the documentation for getting started with Blazor WebAssembly App and Syncfusion Blazor Components with Authentication Library.
+platform: Blazor
+component: Common
+documentation: ug
+---
+
+# Getting Started with Blazor WASM App with Authentication Library
+
+This article provides a step-by-step instructions for building and securing Blazor WebAssembly Standalone App with Blazor WebAssembly Authentication library using [Visual Studio](https://visualstudio.microsoft.com/vs/).
+
+## Prerequisites
+
+* [System requirements for Blazor components](https://blazor.syncfusion.com/documentation/system-requirements)
+
+## Create a new Blazor WebAssembly Standalone App in Visual Studio
+
+You can create a **Blazor WebAssembly Standalone App** using Visual Studio via [Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-8.0&pivots=windows) or the [Syncfusion Blazor Extension](https://blazor.syncfusion.com/documentation/visual-studio-integration/template-studio) by setting the `Authentication type` to `Individual Accounts`. This selection adds authentication support and doesn't result in storing users in a database.
+
+## Configure the application with Google Auth 2.0 OIDC
+
+* Set up `Google OAuth 2.0` authentication. For more information, refer [here](https://support.google.com/cloud/answer/6158849?hl=en)
+
+* Now, replace the `appsettings.json` with the following code which contains code to configure the application with `Google OAuth 2.0`.
+
+{% tabs %}
+{% highlight cshtml tabtitle="appsettings.json" %}
+
+{
+  "Local": {
+    "Authority": "https://accounts.google.com/",
+    "ClientId": "2...7-e...q.apps.googleusercontent.com",
+    "PostLogoutRedirectUri": "https://localhost:5001/authentication/logout-callback",
```

Comment:
@SubathraKaliamoorthy - make it more general insteadof provide the port number.

like,

"PostLogoutRedirectUri": "https://<<Approot Url>>/authentication/logout-callback",

Provide example in comments